overlord/snapstate/component_install_test.go

```diff
@@ -365,7 +365,9 @@ func (s *snapmgrTestSuite) TestInstallComponentPathCompRevisionPresent(c *C) {
 		snapstate.Flags{})
 	c.Assert(err, IsNil)
 
-	verifyComponentInstallTasks(c, compOptIsLocal|compOptRevisionPresent|compOptIsActive|compCurrentIsDiscarded, ts)
+	// note that we don't discard the component here, since the component
```

Comment:
do we need a variant of the test were the discard happens because of the change revision, or will it come later when we support that case in general?